training-maquetado/index.html

```diff
@@ -8,6 +8,15 @@
   <link rel="stylesheet" type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1>
+  <section class="content-layout">
+    <div class="d-flex align-center justify-center title-layout">
```

Comment:
We usually try to contain the styles on just one class, but i like this approach aswel 👍